src/lib/getModelData.ts

```diff
@@ -269,8 +269,13 @@ export async function getItemData({
       } else {
         const { gltf, renderer } = await loadGltf(URL.createObjectURL(contents[model]))
         document.body.removeChild(renderer.domElement)
-        const duration = gltf.animations[0].duration
-        const frames = gltf.animations[0].tracks[0].times.length - 1
+        const animation = gltf.animations[0]
+        const duration = animation.duration
```

Comment:
```suggestion

```
I'd add this down below since the variable is not used to compute anything:
```
 info = {
          sequences: gltf.animations.length,
          duration: animation.duration,
          frames: frames,
          fps: frames / duration
        }
```